src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="header__img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link is-active"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. If these links are placeholders and will be replaced with actual links later, consider using `href="#"` for now.